packages/@aws-cdk/cloudformation-include/test/test-templates/invalid/fn-sub-key-not-in-template-string.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Resources": {
+    "AnotherBucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "AccessControl": { "Fn::Sub": "unusued-bucket-fn-sub-${AFakeResource}" }
```

Comment:
Two things:

1. I would use `BucketName` here, as `AccessControl` has its own requirements. 
2. Just do `"Fn::Sub": "${FakeResource}"`. A test should be minimal, and not contain extra baggage that does not contribute to the test in any way.